py3status/modules/file_status.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""
+Display if a file or dir exists.
+
+Configuration parameters:
+    cache_timeout: how often to run the check
+    color_available: color of format_available
+    color_unavailable: color of format_unavailable
+    format_available: what to display when available
+    format_unavailable: what to display when unavailable
+    path: check if file or dir exists
+
+@author obb, Moritz Lüdecke
+"""
+
+from time import time
+import os
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    color_available = None
+    color_unavailable = None
```

Comment:
@ultrabug I worry about having these config params.  Although the names may be appropriate for this module, there is something nice about sharing names eg color_bad as it keeps the configs consistent.

Maybe you have a nice insight on this